tests/backup/backup_orphaned_delete_test.go

```diff
@@ -0,0 +1,241 @@
+			log.FailOnError(err, "Fetching admin ctx")
+			userBackupScheduleNames, err := GetAllBackupSchedulesForUser(nonAdminUserName, commonPassword)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of fetching backups [%v] of user [%s] from px-admin user", userBackupScheduleNames, nonAdminUserName))
+			log.Infof(fmt.Sprintf("The list of user [%s] backups from px-admin %v", nonAdminUserName, userBackupScheduleNames))
+			for _, userBackupScheduleName := range userBackupScheduleNames {
+				log.InfoD(fmt.Sprintf("Verifying deletion of backup schedule [%s] of non-admin user [%s] from px-admin user", userBackupScheduleName, nonAdminUserName))
+				backupScheduleUid, err := GetScheduleUID(userBackupScheduleName, orgID, ctx)
+				log.FailOnError(err, fmt.Sprintf("Fetching schedule uid for shedule [%s]", userBackupScheduleName))
+				err = DeleteScheduleWithClusterRef(userBackupScheduleName, backupScheduleUid, SourceClusterName, srcClusterUid, orgID, adminCtx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of deleting backup scheudle [%s] of user [%s] from px-admin user", userBackupScheduleName, nonAdminUserName))
+			}
+		})
+
+		Step("Verifying updation of cluster of non-admin user from px-admin", func() {
+			log.InfoD("Verifying updation of cluster of non-admin user from px-admin")
+			adminCtx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-admin ctx")
+			srcClusterConfigPath, err := GetSourceClusterConfigPath()
+			log.FailOnError(err, "Fetching source clusterconfigpath")
+			_, err = UpdateCluster(SourceClusterName, srcClusterUid, srcClusterConfigPath, orgID, invalidCredName, invalidCloudCredUID, adminCtx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of updation of cluster [%v] of user [%s] from px-admin user", SourceClusterName, nonAdminUserName))
+			dstClusterConfigPath, err := GetDestinationClusterConfigPath()
+			log.FailOnError(err, "Fetching destination clusterconfigpath")
+			_, err = UpdateCluster(destinationClusterName, destClusterUid, dstClusterConfigPath, orgID, invalidCredName, invalidCloudCredUID, adminCtx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of updation of cluster [%v] of user [%s] from px-admin user", destinationClusterName, nonAdminUserName))
+		})
+
+		Step(fmt.Sprintf("Verifying  deletion of clusters of non-admin user from px-admin user"), func() {
+			log.InfoD("Verifying  deletion of clusters of non-admin user from px-admin user")
+			adminCtx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-admin ctx")
+			err = DeleteClusterWithUid(SourceClusterName, srcClusterUid, orgID, adminCtx, true)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Deleting cluster %s", SourceClusterName))
+			err = DeleteClusterWithUid(destinationClusterName, destClusterUid, orgID, adminCtx, true)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Deleting cluster %s", destinationClusterName))
+		})
+	})
+	JustAfterEach(func() {
```

Comment:
Are we deleting cloud credentials and backup locations of users? Or is it taken care by admin ctx?

Reply:
Here we are creating one user and all cleanup is handled by that user.